vllm/engine/output_processor/multi_step.py

```diff
@@ -69,6 +68,9 @@ def process_outputs(self, sequence_group: SequenceGroup,
         including freeing finished sequences. It also handles cases where there
         are tokens emitted after the EOS token.
         """
+        # TODO: Add support for async if necessary
+        assert not is_async
```

Comment:
What does this `is_async` mean? Can you please document it?

Reply:
Good idea, added doc